src/query/ast/tests/it/testdata/statement.txt

```diff
@@ -11306,6 +11306,131 @@ Grant(
 )
 
 
+---------- Input ----------
+REVOKE write ON stage s1 FROM a;
+---------- Output ---------
+REVOKE Write ON STAGE s1 FROM USER 'a'@'%'
```

Comment:
There is an additional `USER`. Is it expected?

Reply:
except. default is user, also can be role.